backend/__test__/datafileAndNotFilter.test.ts

```diff
@@ -0,0 +1,109 @@
+import request from "supertest";
+import { expect, describe, it } from "@jest/globals";
+import App from "../src/app";
+import mongoose from "mongoose";
+import { MongoMemoryServer } from "mongodb-memory-server";
+import { checkArrayContainsObjects } from "./helper.test";
+
+// Create app
+const app: App = new App();
+
+describe("Checks if AND + NOT boolean operations works", () => {
+  it('Should return {"status":"200"}', async () => {
+    // Create MongoDB
+    const mongoServer = await MongoMemoryServer.create();
+    await mongoose.connect(mongoServer.getUri());
+    const query = {
```

Comment:
can you maybe use a different word than query? like seed or entity or document. I initially assumed these are database queries

Reply:
I will fix it in a new test related branch.